core/server/themes/list.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Store themes after loading them from the file system
+ */
+var _ = require('lodash'),
+    themeListCache = {};
+
+module.exports = {
+    get: function get(key) {
+        return themeListCache[key];
+    },
+
+    getAll: function getAll() {
+        return themeListCache;
+    },
+
+    set: function set(key, theme) {
+        themeListCache[key] = _.cloneDeep(theme);
+        return themeListCache[key];
+    },
+
+    del: function del(key) {
+        delete themeListCache[key];
+    },
+
+    init: function init(themes) {
```

Comment:
Looks like this function was invented for the tests?
Can't we use `set` and `del` instead?